app/api/v1/activities.py

```diff
@@ -162,13 +162,13 @@ async def get_activity_by_cw_unit_id(
     if unit_with_metadata["marketplaceIdentifier"]:
         activity_filters["and"].append(models.Activity.asset_id == unit_with_metadata["marketplaceIdentifier"])
     else:
-        logger.warning(f"retrieved unit does not contain marketplace identifier. unable to get activity record")
+        logger.warning("retrieved unit does not contain marketplace identifier. unable to get activity record")
         return schemas.ActivityRecordResponse()
 
     activity_filters["and"].append(models.Activity.mode == action_mode)
     activity_filters["and"].append(models.Activity.coin_id == coin_id)
 
-    activities: [models.Activity]
+    activities = [models.Activity]
```

Comment:
this... worked before?

Reply:
it did something probably :-) 🤷

Reply:
i was under the impression that such a line did quite literally nothing (in the sense that it resulted in an unusable variable until initialization), but i didn't want to cause a headache for myself if the author of line 80 knew something i didn't :)